packages/block-editor/src/components/block-list/use-block-props/index.js

```diff
@@ -125,7 +123,7 @@ export function useBlockProps( props = {}, { __unstableIsHtml } = {} ) {
 			isEnabled: isSectionBlock,
 		} ),
 		useScrollIntoView( { isSelected } ),
-		canDrag ? ffDragRef : undefined,
+		canMove ? ffDragRef : undefined,
```

Comment:
This fixes the issue where you can't drag a quote block after selecting a paragraph inside it in trunk.